complex-events/filter_clinvar_xml.py

```diff
@@ -0,0 +1,66 @@
+import argparse
```

Comment:
I'm wondering if we should have a top level directory with analyses or data exploration and sub directories with `complex_events` and `clinvar-variant-types`